sa/type-converter_test.go

```diff
@@ -83,7 +84,8 @@ func TestJSONWebKeyBadJSON(t *testing.T) {
 	scanner, _ := tc.FromDb(&out)
 	err := scanner.Binder(&badJSON, &out)
 	test.AssertError(t, err, "expected error from scanner.Binder")
-	badJSONErr, ok := err.(errBadJSON)
+	var badJSONErr errBadJSON
+	ok := errors.As(err, &badJSONErr)
```

Comment:
Sorry, this one could have been inlined as well.

Reply:
No worries, Thanks for being so thorough!